data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/GetStateFinalityCheckpointsTest.java

```diff
@@ -96,4 +96,12 @@ void metadata_shouldHandle200() throws IOException {
 
     assertThat(data).isEqualTo(expected);
   }
+  @Test
```

Comment:
that's ok, just please mind the gap line between the tests as in the original file